### PR TITLE
SyntacticValidate and cross-network broadcast tests

### LINKED ISSUE #912: [Test] Test syntacticValidate for transfer transactions

### LINKED ISSUE #945: [Test] Transaction cross-network broadcast
Test if a transaction created on a specific network (e.g. Toplnet, Valhalla, Hel) can be broadcast on a different network from the one in which it was created.

### PR DESCRIPTION
**Pull request by Tuxman**
closes #912 
closes #945

### CODE REVIEW

File: src/test/scala/co/topl/utils/ValidGenerators.scala [hunk 1]
```diff
@@ -108,16 +109,75 @@ trait ValidGenerators extends CoreGenerators {
...
+      recipients,
+      IndexedSeq(sender),
+      changeAddress = sender,
+      None,
+      fee,
+      data = None
+    ).get
+
+    val sig = keyRing.signWithAddress(sender)(rawTx.messageToSign).get
+    val tx = rawTx.copy(attestation = Map(prop -> sig))
```
Comment (scasplte2) [inline src/test/scala/co/topl/utils/ValidGenerators.scala:132]: @Tuxman I don't think you need to change this, just FYI there should be a function `updateAttestation` that does this bit for you.

**Review by scasplte2 — COMMENTED** [review]
@Tuxman I don't quite understanding what is happening here. Are you using these new generators anywhere?

**Comment by scasplte2** [comment]
so what is the plan with this @Tuxman

**Comment by Tuxman** [comment]
> so what is the plan with this @Tuxman

I am going to redo the tests for this today.

**Pull request merged**